proxy/src/main/java/com/velocitypowered/proxy/config/VelocityConfiguration.java

```diff
@@ -525,6 +532,7 @@ public static VelocityConfiguration read(Path path) throws IOException {
         PlayerInfoForwarding.NONE);
     PingPassthroughMode pingPassthroughMode = config.getEnumOrElse("ping-passthrough",
         PingPassthroughMode.DISABLED);
+    boolean samplePlayersInPing = config.getOrElse("sample-players-in-ping", true);
```

Comment:
Imo the default should be `false` to maintain the current behavior; lots of users already use a plugin to override the player list and enabling this is wasteful since the plugin overrides the list anyways.